Invoke-BeforePack.ps1

```diff
@@ -0,0 +1,19 @@
+# This script tries to work around NuGet's validation and the NU5100 warning caused by the presence of these files:
+# - bin/Release/netstandard2.0/Lombiq.Analyzers.dll
+# - obj/Release/netstandard2.0/Lombiq.Analyzers.dll
+# The cause is probably that we don't actually do anything in the corresponding csproj, it only exists because we need
+# an entry point for `dotnet pack` (unlike the older `nuget pack` tooling, which worked with just the .nuspec alone).
+
+$release = Get-ChildItem Lombiq.Analyzers/bin/Release -Recurse -Include Lombiq.Analyzers.dll
+$releasePdb = $release.FullName -replace 'dll$','pdb'
+New-Item -Type Directory -Force lib
+Copy-Item $release lib
```

Comment:
It should, but it doesn't.

Reply:
We can't drop it, because of the other half of the comment (NuGet's validation). So yeah apparently it doesn't help with NU5100 warning, I guess I forgot to clean up the comment. I did so now.

I dread to touch this file, because if I mess something up I won't be able to see for myself. I'd have to wait and ask Zoltán if he got one of these accursed mails or not.
![image](https://user-images.githubusercontent.com/4007293/182733253-940309b0-939b-4970-8246-2f1f18fe15af.png)
What I have above I reached purely by trial and error using this bs feedback. So I know it works and it's necessary. Don't know why, because NuGet itself is an opaque web service and web2 was a mistake.

Reply:
I feel your pain! I just saw that you're copying the .dll and .pdb files into _lib_ directly, **not** into any **framework-specific folder** under _lib_, which is what the link above suggests. From all the documentation on NU5100 says on the topic, that should fix the warning which would be nice™! I know you're faaar beyond nice with this issue, but suppressing warnings when we don't have to wouldn't be good in the long run.

I'm trying to fix this right now.

Reply:
Turns out that removing the .dll files from _\bin_ and _\obj_ folders fixes the NU5100 warning, so I was able to remove that from the GHA `publish-nuget` action. 🎉 

Now, creating a symbol package for a package that doesn't contain any .dll tripped the Symbol package validation process at NuGet.org:
![image](https://user-images.githubusercontent.com/164360/182841966-ec836c09-7944-4e62-9e82-9d55c14c5d34.png)

This could be fixed by not publishing a symbol package in the first place. Which seems to make sense, given that `Lombiq.Analyzers` doesn't contain any code.

To support that in our publish workflow, we'd need a new parameter along the lines `SkipSymbolPackage` that would default to `false`, which we could set to `true` for the Analyzers package.

What do you think @DAud-IcI and @Piedone?

Reply:
I'm OK with that, just make sure to document in the YAML of this project why we're doing this.

Reply:
That worked: https://github.com/Lombiq/.NET-Analyzers/runs/7672221377.